eb

```diff
@@ -44,43 +44,34 @@ REQ_MAJ_PYVER=2
 REQ_MIN_PYVER=6
 REQ_PYVER=${REQ_MAJ_PYVER}.${REQ_MIN_PYVER}
 
+PYTHON=python$REQ_MAJ_PYVER
+which $PYTHON &> /dev/null
+if [ $? -ne 0 ]
+then
+    PYTHON=python
+    which $PYTHON &> /dev/null
+    if [ $? -ne 0 ]
+    then
+        echo "ERROR: $PYTHON not available in \$PATH?"
+        exit 1
+    fi
+fi
+
 # make sure Python version being used is compatible
-pyver=`python -V 2>&1 | cut -f2 -d' '`
+pyver=`$PYTHON -V 2>&1 | cut -f2 -d' '`
 pyver_maj=`echo $pyver | cut -f1 -d'.'`
 pyver_min=`echo $pyver | cut -f2 -d'.'`
 
 if [ $pyver_maj -ne $REQ_MAJ_PYVER ]
 then
     echo "ERROR: EasyBuild is currently only compatible with Python v${REQ_MAJ_PYVER}.x, found v${pyver}" 1>&2
-    exit 1
+    exit 2
```

Comment:
but you're changing the meaning of the exit codes, scripts that relied on them will start acting weird? 
so it's better to keep the meaning of the codes the same, or just always exit 1 since the meaning can change arbitrarily anyway?